src/treeview.ts

```diff
@@ -243,6 +244,7 @@ class MetalsTreeDataProvider implements TreeDataProvider<string> {
         }
       }
     }
+    return new ThemeIcon(icon);
```

Comment:
If `icon` is an existing VS Code icon it will now show up here. We could probably remove some icons and replace with existing ones, but I think it's fine as is now.